pkg/envConfig/config.go

```diff
@@ -21,6 +23,13 @@ type PDConfig struct {
 	MySubdomain []Subdomain `json:"subdomains"`
 }
 
+// cloudtrailCmd configuration struct for parsing configuration options
+type Configuration struct {
```

Comment:
**Question:** if this is the `Configuration`, what is the other stuff? It's all configuration and probably should be in the same struct. The load configuration function should load the configuration for all of it. 

We don't need to change the whole configuration loading for now, but let's rename it to CloudTrailConfig.